src/wallet/wallet.cpp

```diff
@@ -146,6 +146,15 @@ std::shared_ptr<CWallet> LoadWallet(interfaces::Chain& chain, const WalletLocati
     }
     AddWallet(wallet);
     wallet->postInitProcess();
+
+    {
+        // Update wallet transactions with current mempool transactions.
+        LOCK2(cs_main, ::mempool.cs);
+        for (const CTxMemPoolEntry& entry : ::mempool.mapTx) {
```

Comment:
The wallet module is not allowed to call directly into the node. This should be an interface method.

Reply:
Right, added 51cb2eb and then refactored `LoadWallet` to use it.